README.md

```diff
@@ -1,59 +1,56 @@
-Heureka ověřeno PHP API
-=======================
+Heureka 'Ověřeno zákazníky' PHP API
+===================================
 
-[Heureka Ověřeno](http://overeno.heureka.cz/) service API for PHP. 
+[Heureka Ověřeno zákazníky](http://overeno.heureka.cz/) (ShopCertification) service API client implementation for PHP.
+
+Examples
+--------
+
+You can check working examples in the folder `examples` of this repository.
 
 Usage
 -----
 
-Initialize Service using [your API key](http://sluzby.heureka.cz/sluzby/certifikat-spokojenosti/):
+Initialize class `Heureka\ShopCertification` using
+[your API key](http://sluzby.heureka.cz/sluzby/certifikat-spokojenosti/):
```

Comment:
dopsal bych pred odkaz neco jako "you can get your key **after login** at http://"